src/main/java/com/maks/durov/amazonreviewapp/entity/Review.java

```diff
@@ -0,0 +1,25 @@
+package com.maks.durov.amazonreviewapp.entity;
+
+import java.time.LocalDateTime;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import lombok.Data;
+
+@Data
+@Entity(name = "reviews")
+public class Review {
+    @Id
+    private Long id;
+    private String summary;
+    @Column(length = 100000)
+    private String text;
+    private LocalDateTime reviewTime;
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    private Product product;
```

Comment:
CascadeType.ALL: you will delete one Review and Product also will be deleted.

Reply:
got it